app/views/price_groups/_users.html.haml

```diff
@@ -1,24 +1,9 @@
 %p Pricing policies apply to orders placed by the the following users:
```

Comment:
Since we're here changing this file, can you pull all the copy text into the translation file?